config/betterquesting/DefaultQuests/Quests/Tier5IV-AAAAAAAAAAAAAAAAAAAABw==/MoreSamarium-u_-8t2IvRnydvmUx_qdYuA==.json

```diff
@@ -29,7 +29,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "If you have a stable samarium mining resource acquisition, refining samarium ore resource should be the best way to get samarium.\n\nFirst step prepare Samarium ore concentrate dust or crushed Samarium ore(you should not have this now) , use nitric acid digest ore resource in Digester, get Muddy Samarium Rare Earth Solution. The Thorium and Phosphorus are retained in the form of solid slag powder. \n\nThen transfer to a Dissolution Tank and continued to soak in nitric acid. The undissolved powder is then meticulously filtered out, mainly the Cerium resource.\n\nThen dilute with water to adjust the pH."
+      "desc:8": "If you have a stable Samarium resource acquisition, refining Samarium Ore should be the best way to get Samarium.\n\nAs the first step, disolve some Samarium Ore Concentrate Dust or Crushed Samarium Ore (you probably don\u0027t have access to this option yet) in Nitric Acid in a Digester to get Muddy Samarium Rare Earth Solution. Thorium and Phosphate are retained in the form of a solid slag powder. \n\nThen transfer the muddy solution to a Dissolution Tank and continued to soak in nitric acid. The undissolved powder is then meticulously filtered out, yielding some Cerium.\n\nThen dilute the result with water to adjust the pH."
```

Comment:
"and continued to soak" sounds off - perhaps "and soak it in nitric"

Reply:
just a d too much I woul say